frontend/src/tests/import-export.test.js

```diff
@@ -1,8 +1,31 @@
 /**
  * @jest-environment node
  */
+/* eslint-env node */
 import { describe, it, expect } from "./utils";
-import { validate, SpreadsheetRowMapper } from "../libs/importExportUtils";
+import {
+    validate,
+    SpreadsheetRowMapper,
+    prepareSpreadsheet,
+} from "../libs/importExportUtils";
+import {
+    instructorData,
+    applicantData,
+    positionData,
+    assignmentData,
+    ddahData,
+} from "./import-export-data/export-data";
+import { prepareMinimal } from "../libs/exportUtils";
+// BELOW ARE USED FOR ROUND TRIP TEST FOR prepareData
+// import { prepareData } from "../views/instructors/import-export";
+// import XLSX from "xlsx";
+
+// function File(fileBits, fileName, options) {
+//     this.fileBits = fileBits;
+//     this.fileName = fileName;
+//     this.options = options;
+// }
+// global.File = File;
```

Comment:
Why are these comments here?

Reply:
These are only used for round trip test, which currently has some issues

Reply:
removed round trip test for this PR